packages/shared/src/utils/constants/translation-flag.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import {
+    FlagArabLeagueIcon,
+    FlagBangladeshIcon,
+    FlagChinaSimplifiedIcon,
+    FlagChinaTraditionalIcon,
+    FlagFranceIcon,
+    FlagGermanyIcon,
+    FlagIndonesiaIcon,
+    FlagItalyIcon,
+    FlagMongoliaIcon,
+    FlagPolandIcon,
+    FlagPortugalIcon,
+    FlagRussiaIcon,
+    FlagSpainIcon,
+    FlagSriLankaIcon,
+    FlagThailandIcon,
+    FlagTurkeyIcon,
+    FlagUnitedKingdomIcon,
+    FlagVietnamIcon,
+} from '@deriv/quill-icons';
+
+type TTranslationFlag = {
+    [key: string]: (size?: number) => React.ReactNode;
+};
+
+export const TranslationFlag: TTranslationFlag = {
+    AR: size => <FlagArabLeagueIcon width={size} height={size} />,
```

Comment:
width and height are equal for flags. isnt?
how about  iconSize prop instead of width and height? will it work?

Reply:
sureee will try iconSize instead 🙏